qdrant-landing/content/articles/qdrant-1.3.x.md

```diff
@@ -0,0 +1,200 @@
+
+```python
+storage:
+	# enable the async scorer which uses io_uring
+	async_scorer: true
+```
+You can return to the mmap based backend by either deleting the `async_scorer` entry or setting the value to `false`.
+
+This operation generates a lot of disk IO, so it is a prime candidate for possible improvements.
+Please keep in mind that this feature is experimental and that the interface may change in further versions.
+
+### Grouping API lookup
+
+In version 1.2.0, we introduced a mechanism for grouping requests. Our new feature extends this functionality by giving you the option to look for points in another collection using group ids. We wanted to add this feature, since having multiple points for parts of the same item optimizes storage use, particularly if the payload is large.
+
+#### Adding the parameter to grouping API request:
+
+When using the [**grouping** API](#grouping-api), add the `with_lookup` parameter to bring the information from those points into each group:
+
+```http
+POST /collections/chunks/points/search/groups
+{
+    // Same as in the regular search API
+    "vector": [1.1],
+    ...,
+    // Grouping parameters
+    "group_by": "document_id",  
+    "limit": 2,                 
+    "group_size": 2,            
+    // Lookup parameters
+    "with_lookup": {
+        // Name of the collection to look up points in
+        "collection_name": "documents",
+        // Options for specifying what to bring from the payload 
+        // of the looked up point, true by default
+        "with_payload": ["title", "text"],
+        // Options for specifying what to bring from the vector(s) 
+        // of the looked up point, true by default
+        "with_vector: false,
+    }
+}
+```
+
+```python
+client.search_groups(
+    collection_name="chunks",
+
+    # Same as in the regular search() API
+    vector=[1.1],
+    ...,
+
+    # Grouping parameters
+    group_by="document_id", # Path of the field to group by
+    limit=2,                # Max amount of groups
+    group_size=2,           # Max amount of points per group
+
+    # Lookup parameters
+    with_lookup=models.WithLookup(
+        # Name of the collection to look up points in
+        collection_name="documents",
+
+        # Options for specifying what to bring from the payload 
+        # of the looked up point, True by default
+        with_payload=["title", "text"]
+
+        # Options for specifying what to bring from the vector(s) 
+        # of the looked up point, True by default
+        with_vector=False, 
+    )
+)
+```
+
+This has the extra benefit of having a single point to update when the information shared by the points in a group changes. For example, if you have a collection of documents, you may want to chunk them and store the points for the chunks in a separate collection, making sure that you store the point id from the document it belongs in the payload of the chunk point.
```

Comment:
It may be nice to add a diagram to show the documents in one collection, and the respective chunks in another one

Reply:
definitely - do you mind if I add yours?

Reply:
Go ahead. It can also be the actual collection setup, where one point in one collection is related to multiple ones in the other.